server/remote_cache/hit_tracker/hit_tracker.go

```diff
@@ -34,6 +36,21 @@ const (
 	// New counter types go here!
 )
 
+const (
+	// Prometheus label values
+
+	// CacheEventTypeLabel values
+
+	hitLabel    = "hit"
+	missLabel   = "miss"
+	uploadLabel = "upload"
+
+	// CacheTypeLabel values
+
+	actionCacheLabel = "action_cache"
+	casLabel         = "cas"
+)
```

Comment:
You can just put this stuff in the const block above line 21?

Nit: remove newlines after comments

Reply:
Moved them above line 21 -- kept the newlines though -- I added the newlines so they aren't syntactically part of the documentation for the const immediately following them. e.g. if you hover over "hitLabel" in your IDE or auto-generated godocs or something, it would show the section comment, which is not really intended